cpp/ql/test/library-tests/ir/ir/aliased_ir.expected

```diff
@@ -2991,7 +2991,7 @@ generic.c:
 #   14|     r14_1(glval<int>)     = VariableAddress[i]       : 
 #   14|     m14_2(int)            = Uninitialized[i]         : &:r14_1
 #   16|     r16_1(glval<char *>)  = VariableAddress[#return] : 
-#   16|     r16_2(glval<char[4]>) = StringConstant[int]      : 
+#   16|     r16_2(glval<char[4]>) = Constant[int]            : 
```

Comment:
This is due to the the `getType` on the `Value` associated with the Generic being of `ArrayType` (only the underlying type is). This seems inconsequential though.